config/crd/bases/datasciencepipelinesapplications.opendatahub.io_datasciencepipelinesapplications.yaml

```diff
@@ -106,6 +106,16 @@ spec:
                     description: 'Default: true Deprecated: DSP V1 only, will be removed
                       in the future.'
                     type: boolean
+                  customKfpLauncherConfigMap:
+                    description: CustomKfpLauncherConfig is a custom config file that
+                      you can provide for the api server to use instead of the one
```

Comment:
this is subtle and tricky, but it's not apiserver that reads the kfp-launcher configmap. It's the kfp-launcher and driver binaries that read it. Those are technically not part of the apiserver.

Reply:
@HumairAK did I get this wrong? apiserver *does* read it? (from our office hours meeting last week...)

Reply:
the driver reads the configmap when setting the context property for the context.pipelinerun 
the kfp-launcher reads the context property from mlmd